backend/backend/core/agents.py

```diff
@@ -1,22 +1,27 @@
 from langchain_core.prompts import ChatPromptTemplate
+from langgraph.errors import GraphRecursionError
```

Comment:
Added unnecessary import from langgraph.errors. Consider removing it if not used.